code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -494,8 +499,11 @@ var/list/ai_list = list()
 	if(Bot.calling_ai && Bot.calling_ai != src) //Prevents an override if another AI is controlling this bot.
 		src << "<span class='danger'>Interface error. Unit is already in use.</span>"
 		return
-
+	src << "<span class='notice'>Sending command to bot...</span>"
+	call_bot_cooldown = world.time + CALL_BOT_COOLDOWN
 	Bot.call_bot(src, waypoint)
+	call_bot_cooldown = FALSE

```

Comment:
yes, we do, but i don't want to fuck a traitor ai out the ability to control pingsky for the rest of the round because astar is being a meme